src/main/webapp/WEB-INF/category.jsp

```diff
@@ -52,7 +52,7 @@
     <button onclick="getLocation()">Add your location</button>
     <div id="map"></div>
     <% }  %>
-    <div id="message-container">
+    <div class="message-container">
```

Comment:
I'm curious, what does this change do?

Reply:
It doesn't make a difference in terms of functionality I believe. I think class is for repeated divs, and id is for one-time-only type divs? Not too sure though.